tests/test_bodies.py

```diff
@@ -16,37 +16,45 @@ def test_body_from_parameters_raises_valueerror_if_k_units_not_correct():
     _name = _symbol = ""
     _R = 0
     with pytest.raises(u.UnitsError) as excinfo:
-        bodies.Body.from_parameters(None, wrong_k, _name, _symbol, _R)
+        Body.from_parameters(None, wrong_k, _name, _symbol, _R)
     assert (
         "UnitsError: Argument 'k' to function 'from_parameters' must be in units convertible to 'km3 / s2'."
         in excinfo.exconly()
     )
 
 
+def test_body_from_parameters_returns_body_object():
+    k = 3.98e5 * u.km ** 3 / u.s ** 2
+    _name = _symbol = "jupiter"
+    jupiter = Body.from_parameters(Sun, k, _name, _symbol, Jupiter.R)
```

Comment:
I find this test a bit weird. What about checking that `k` has the given value, for example?

Reply:
Thanks for this suggestion! The motivation for this test was from here: https://codecov.io/gh/poliastro/poliastro/src/main/src/poliastro/bodies.py at line 105, i.e. to check whether `from_parameters` works properly. I checked for both `k` and `R` here.

Do you think we should use this test for all bodies? I thought checking for any one body would suffice for the purposes of this test.